src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -46,15 +50,28 @@ public EditCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_NAME).isPresent()) {
             editPersonDescriptor.setName(ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get()));
         }
+        if (argMultimap.getValue(PREFIX_GENDER).isPresent()) {
+            editPersonDescriptor.setGender(ParserUtil.parseGender(argMultimap.getValue(PREFIX_GENDER).get()));
+        }
         if (argMultimap.getValue(PREFIX_PHONE).isPresent()) {
             editPersonDescriptor.setPhone(ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get()));
         }
         if (argMultimap.getValue(PREFIX_EMAIL).isPresent()) {
             editPersonDescriptor.setEmail(ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get()));
         }
+        if (argMultimap.getValue(PREFIX_MEASUREMENT).isPresent()) {
+            editPersonDescriptor.setMeasurement(ParserUtil.parseMeasurement(
+                   argMultimap.getValue(PREFIX_MEASUREMENT).get())
+            );
+        }
```

Comment:
I'm not sure if this is actually a bug; but since you use the parseMeasurement without the gender here, I think it might be possible for someone to edit an existing Male in the database, and replace their measurements with a females?